Tools/Cli-LoRa-Device-Provisioning/LoRaWan.Tools.CLI/Program.cs

```diff
@@ -251,8 +256,8 @@ private static async Task<ConfigurationContent> GetEdgeGatewayDeployment(AddGate
             var tokenReplacements = new Dictionary<string, string>
             {
                 { "[$reset_pin]", opts.ResetPin.ToString() },
-                { "[\"$spi_speed\"]", opts.SpiSpeed != AddGatewayOption.DefaultSpiSpeed ? string.Empty : ",\"SPI_SPEED\":{\"value\":\"2\"}" },
-                { "[\"$spi_dev\"]", opts.SpiDev != AddGatewayOption.DefaultSpiDev ? string.Empty : $",\"SPI_DEV\":{{\"value\":\"{opts.SpiDev}\"}}" },
+                { "[\"$spi_speed\"]", opts.SpiSpeed == AddGatewayOption.DefaultSpiSpeed ? string.Empty : ",\"SPI_SPEED\":{\"value\":\"2\"}" },
```

Comment:
Are the change of conditions here expected?

Reply:
Yes, the tests help me discover the bug. But let me validate with you.

if spi-speed == 8
   do not set property
else
  set property to value 2

if spiDev == 0
  do not set property
else
  set property to value spiDev